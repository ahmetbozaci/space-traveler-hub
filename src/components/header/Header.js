/** @format */

import { Navbar, Container, Nav } from 'react-bootstrap';
import { Link } from 'react-router-dom';
import planet from '../../planet.png';

const Header = () => (
  <>
    <Navbar bg="light">
      <Container>
        <Navbar.Brand href="#home">
          <img
            src={planet}
            width="30"
            height="30"
            className="d-inline-block align-top"
            alt="React Bootstrap logo"
          />
          Space Travelers Hub
        </Navbar.Brand>
        <Nav>
          <Nav.Link>
            <Link to="/rockets"> Rockets </Link>
          </Nav.Link>
          <Nav.Link>
            <Link to="/missions"> Missions </Link>
          </Nav.Link>
          <Nav.Link>
            <Link to="/my-profile"> My Profile </Link>
          </Nav.Link>
        </Nav>
      </Container>
    </Navbar>
  </>
);

export default Header;
