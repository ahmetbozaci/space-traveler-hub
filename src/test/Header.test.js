import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/header/Header';

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
