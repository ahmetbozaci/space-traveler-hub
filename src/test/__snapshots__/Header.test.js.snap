// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<nav
  className="navbar navbar-light bg-light"
>
  <div
    className="container-fluid"
  >
    <span
      className="navbar-brand"
    >
      <img
        alt="React Bootstrap logo"
        className="d-inline-block align-top"
        height="30"
        src="planet.png"
        width="30"
      />
      Space Travelers Hub
    </span>
    <button
      aria-controls="offcanvasNavbar"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
  </div>
</nav>
`;
